### ISSUE TITLE
[General Question] Accessibility Issues

### ISSUE THREAD

**Issue by delphinegilbert** [microsoft/accessibility-insights-windows#1738]
Hello,

An accessibility audit was performed on Accessibility Insights for Windows in our department and the following issues were found:

**Issue 1 Using Placeholder text as a label**

**Description of issue**

At the top of the screen, right under "Live Inspect", there is a search field. The instructions "Search by name and control type" are provided to users as placeholder text in the search field, which disappears when you start typing in.

**Impact on users**

This will especially impact users with cognitive challenges such as short term memory loss, but can potentially impact any type of users. Instructions for an input field should be present at all time as a label generally above or next to the field to allow users to keep referring to it while they fill out the input field.

**Steps to reproduce**

Launch the program, and dismiss the startup windows. Using your mouse, hover or click on any other window for the program to start inspecting. It should then display an item tree, there there's a search field at the top.

![image](https://github.com/microsoft/accessibility-insights-windows/assets/59966346/cc547773-96a3-4165-ac6d-1845a82d88a4)

**Issue 2 Non-text elements lack sufficient colour contrast.**

**Description of issue**

Right under "Live Inspect", there is a search input field that lacks any kind of visible border which makes it difficult to see. The input field's colour contrast fails against the white background.

**Impact on users**

This will especially impact people with low vision and non-keyboard, as users may not see at all that they the input field is there.

**Steps to reproduce**

Launch the program, and dismiss the startup windows. Using your mouse, hover or click on any other window for the program to start inspecting. It should then display an item tree, there there's a search field at the top.

![image](https://github.com/microsoft/accessibility-insights-windows/assets/59966346/d77bf17b-27ff-4d09-b518-b9e4ea714601)

**Issue 3: New or changed content not being conveyed to Assistive Technologies**

**Description of issue**

When shifting the separator bar left or right to reorganize the view using the keyboard, there is no feedback to assistive technologies that the action is successfully being performed.

**Impact on users**

This would have a limited impact on users; only sighted users would want to reorganize the view using this separator. Sighted users typically don't need assistive technologies to help them identify what's being done visually on the screen.

However, there are exceptions. For example, a sighted user with attention deficit might use a screen reader to help them keep their focus on what's going on on screen. In this scenario, such person might rely on screen reader feedback to let them know that they are changing the view - and how much (left / right) - in case they get distracted and are doing so none-intentionally.

That said, it would be fair to assume that the feature isn't a crucial one as it is mostly for esthetic. This issue isn't a show stopper.

**Steps to reproduce**

Launch Accessibility Insights, dismiss the opening dialog. You should see the main screen with the separator bar separating "Live Inspect" from the tab "Details". Using your keyboard, navigate to the separator bar and shift it's position left or right using the arrow keys.

![image](https://github.com/microsoft/accessibility-insights-windows/assets/59966346/0a80f7ed-e530-4cdd-8e2e-caa119ce7dbc)

**Comment by v-prachin** [comment]
Acknowledged the issue and looking into it.

**Comment by JGibson2019** [comment]
@delphinegilbert thank you for bringing these to our attention! We'd like to request that you open a separate issue for each of the three that you've listed here and then close out this issue once you've done so. This will allow us to triage and provide additional contextualized discussion.

**Comment by olgatolk71** [comment]
Hi @JGibson2019, I am acting technical advisor and working with Delphine, could you please specify what do you meant by opening a separate issues. Is there a specific platform for that?

**Comment by JGibson2019** [comment]
@olgatolk71 Rather than open one issue for three things, we're requesting that you open one issue in this repo for each of the issues Delphine identified. The platform would still be GitHub. Hope that helps!

**Comment by v-viyada** [comment]
Hi @delphinegilbert / @olgatolk71 , Have you got chance to look and split this issue in 3 separate issues.

**Comment by DaveTryon** [comment]
@delphinegilbert, I've split this into 3 issues and am closing this one out. Thanks!
